src/main/java/org/springframework/data/convert/CustomConversions.java

```diff
@@ -902,9 +946,30 @@ public ConverterConfiguration(StoreConversions storeConversions, List<?> userCon
 		public ConverterConfiguration(StoreConversions storeConversions, List<?> userConverters,
 				Predicate<ConvertiblePair> converterRegistrationFilter) {
 
+			this(storeConversions, userConverters, converterRegistrationFilter, new SimplePropertyValueConversions());
+		}
+
+		/**
+		 * Create a new ConverterConfiguration holding the given {@link StoreConversions} and user defined converters as
+		 * well as a {@link Collection} of {@link ConvertiblePair} for which to skip the registration of default converters.
+		 * <br />
+		 * This allows store implementations to modify default converter registration based on specific needs and
+		 * configurations. User defined converters will are never subject of filtering.
+		 *
+		 * @param storeConversions must not be {@literal null}.
+		 * @param userConverters must not be {@literal null} use {@link Collections#emptyList()} instead.
+		 * @param converterRegistrationFilter must not be {@literal null}.
+		 * @param propertyValueConversions can be {@literal null}.
+		 * @since 2.7
+		 */
+		public ConverterConfiguration(StoreConversions storeConversions, List<?> userConverters,
+				Predicate<ConvertiblePair> converterRegistrationFilter,
+				@Nullable PropertyValueConversions propertyValueConversions) {
```

Comment:
Having a null-object for `PropertyValueConversions` would allow to avoid nullable constructor args.